src/integration_test/java/com/example/Restaurant_Management_System_REST_API/rest/controller/ReservationControllerIntegrationTest.java

```diff
@@ -90,11 +97,25 @@ public void prepareRestaurantClient() {
         customerRepository.save(restaurantCustomer);
     }
 
+    @BeforeEach
+    public void prepareRestaurantTables() {
+        // Create tables with initial values
+        Table table1 = new Table(null, true, null, new ArrayList<>());
+        Table table2 = new Table(null, true, null, new ArrayList<>());
+        Table table3 = new Table(null, true, null, new ArrayList<>());
+        tableList = Arrays.asList(table1, table2, table3);
+
+        // Save each table in the repository
+        for (Table table: tableList) {
+            tableRepository.save(table);
+        }
```

Comment:
okay great! this time I have chosen for loop but Stream would be also great!

Reply:
I didn't mean to use Stream instead of a loop. I wanted to point out that you'd like to use a loop to do all of those much simpler. There are 3 exact same lines of code but assigned to different variables. It can be improved in that way!